src/utils/tagged-template-literal.js

```diff
@@ -15,21 +15,35 @@ const hasInterpolations = node => !node.quasi.quasis[0].tail
  * Merges the interpolations in a parsed tagged template literals with the strings
  */
 const interleave = (quasis, expressions) => {
+  // Used for making sure our dummy mixins are all unique
+  let count = 0
   let css = ''
   for (let i = 0, l = expressions.length; i < l; i += 1) {
     const prevText = quasis[i].value.raw
     const nextText = quasis[i + 1].value.raw
-    const expression = expressions[i]
 
     css += prevText
-    if (isLastLineWhitespaceOnly(prevText) && !isEmptyOrSpaceOnly(prevText)) {
-      css += `-styled-mixin: ${expression.name}`
-      if (nextText.charAt(0) !== ';') {
-        css += ';'
+    let substitute
```

Comment:
I know this isn't actually necessary, but I was in the middle of adding it because I was erroneously recreating the css variable because I wanted to access all the previously processed text, but as I realized the `css` variable already did this I thought it was quite clean with the `substitute` variable and just left it there.